### PR TITLE
Package cpp-half

### PR DESCRIPTION
**Pull request by davidbrochart**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@JohanMabille @SylvainCorlay do you agree to be maintainers of this package?

### CODE REVIEW

File: recipes/cpp-half/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
+  host: []  # Empty host dependency section
```
Comment (scopatz) [inline recipes/cpp-half/meta.yaml:22]: Please just remove the host entry

File: recipes/cpp-half/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    half-precision floating-point type along with corresponding arithmetic
+    operators, type conversions and common mathematical functions. It aims for
+    both efficiency and ease of use, trying to accurately mimic the behaviour
+    of the built-in floating-point types at the best performance possible.
+
+extra:
+  recipe-maintainers:
+    - davidbrochart
+    - SylvainCorlay
+    - JohanMabille
```
Comment (scopatz) [inline recipes/cpp-half/meta.yaml:47]: CC @SylvainCorlay @JohanMabille - are you willing to be maintainers?

**Comment by JohanMabille** [comment]
I confirm that I will be a maintainer.

File: recipes/cpp-half/meta.yaml (hunk 2 above)
Comment (SylvainCorlay) [reply recipes/cpp-half/meta.yaml:48]: yes

**Comment by scopatz** [comment]
Let me know when this is ready!

**Comment by davidbrochart** [comment]
It's ready @scopatz !

**Pull request merged**